ci-operator/step-registry/origin/e2e-aws-sdn-multi/origin-e2e-aws-sdn-multi-workflow.yaml

```diff
@@ -0,0 +1,14 @@
+workflow:
+  as: origin-e2e-aws-sdn-multi
+  steps:
+    pre:
+    - chain: ipi-conf-aws
+    - ref: sdn-conf
+    - ref: sdn-conf-multi-manifest
+    - chain: ipi-install
+    test:
+    - ref: origin-e2e-test
+    post:
+    - chain: ipi-deprovision
+  documentation: |-
+    The Origin E2E SDN workflow executes the common end-to-end test suite with the OpenShiftSDN multitenant network plugin.
```

Comment:
nit - here in the documentation change "The Origin E2E SDN workflow" -> "The Origin E2E SDN multi workflow" this will differentiate the doc string from the single tenet case

Reply:
Changed